src/components/userInfo/index.js

```diff
@@ -1,17 +1,16 @@
 import React, {Component} from 'react';
 import {HeadingThree} from "../headingThree";
 import {Paragraph} from "../paragraph";
-import axios from "axios";
 import './index.css';
 
 export class UserInfo extends Component {
 
-
     render() {
         return (
-            <div className="userInfo">
```

Comment:
as you've removed this class name surely the CSS won't work

Reply:
have applied the class in the level above instead do not panic.